src/backend/launcher.ts

```diff
@@ -737,6 +737,13 @@ async function callRunner(
         rej('Process terminated with signal ' + signal)
       }
 
+      if (options?.runnerStdoutLog) {
+        const f = options.runnerStdoutLog
+        appendFileSync(f, `[${new Date().toLocaleString()}] ${safeCommand}\n`)
+        appendFileSync(f, stdout.join('\n') + '\n\n')
+        appendFileSync(f, stderr.join('\n') + '\n\n')
```

Comment:
...and these in the `data` callback above

Two reasons:
- If the Child Process is terminated/fails for some other reason, the `rej` above makes us skip writing the log
- All stdout will always be written after stderr, which might make troubleshooting more difficult

Reply:
done